Main.ino

```diff
@@ -0,0 +1,27 @@
+/* this is the main file for the interactive light project  and will be uploaded to each arduino
+the code that animate the light is saved in a separate file called lights.h
```

Comment:
Only thing I would change here is lights.h and sensors.h to **lights.ino** and **sensors.ino** since you're referencing the lights.ino page not lights.h